help/de/Help_ch_Account-Actions.xml

```diff
@@ -701,90 +803,121 @@
-                  ist <application>Finance::Quote</application> bereits installiert.
-                </para>
-              </listitem>
+              <varlistentry>
+                <term>&mac;</term>
 
-              <listitem>
-                <para>Trifft keine der obigen Anleitungen auf Ihre Umgebung zu, so können Sie die folgende alternative,
-                  allgemeinere Anweisungen probiern:
-                </para>
+                <listitem>
+                  <para>Wenn Sie &app; mit Hilfe des Installationsprogramms auf der &app; Homepage installiert haben, ist
+                    &app-fq; bereits installiert.
+                  </para>
+                </listitem>
+              </varlistentry>
 
-                <orderedlist spacing="compact">
-                  <listitem>
-                    <para>Schließen Sie alle &app; Anwendungen, die Sie gerade ausführen.
-                    </para>
-                  </listitem>
+              <varlistentry>
+                <term></term>
 
-                  <listitem>
-                    <para>Suchen Sie den Ordner, in dem &app; installiert ist, indem Sie nach <quote>gnc-fq-update</quote>
-                      (ohne die Anführungszeichen) suchen.
-                    </para>
-                  </listitem>
-
-                  <listitem>
-                    <para>Wechseln Sie in dieses Verzeichnis, öffnen Sie eine Root-Shell und führen Sie den Befehl
-                      <code>gnc-fq-update</code> aus. Dadurch wird eine Perl CPAN-Update-Sitzung
-                      gestartet, die ins Internet geht und das Modul
-                      <application>Finance::Quote</application> und seine Abhängigkeiten auf Ihrem
-                      System installiert. Das Programm gnc-fq-update ist interaktiv, bei den meisten
-                      Systemen sollten Sie jedoch die erste Frage mit <quote>Nein</quote>
-                      beantworten können: <quote>Sind Sie bereit für die manuelle Konfiguration?
-                      [ja]</quote> und die Aktualisierung wird automatisch fortgesetzt von diesem
-                      Punkt an.
-                    </para>
-                  </listitem>
-                </orderedlist>
-              </listitem>
-            </itemizedlist>
+                <listitem>
+                  <para>Trifft keine der obigen Anleitungen auf Ihre Umgebung zu, so können Sie die folgende alternative,
+                    allgemeinere Anweisungen probiern:
+                  </para>
 
-            <para>Nachdem die Installation abgeschlossen ist, sollten Sie das Testprogramm <code>gnc-fq-check</code>,
-              das mit &app; verteilt wird, im gleichen Verzeichnis ausführen, um zu testen, ob
-              <application>Finance::Quote</application> installiert ist und richtig funktioniert.
+                  <orderedlist spacing="compact">
```

Comment:
Beachte das Beispiel in https://tdg.docbook.org/tdg/4.5/procedure.html

Scheint mir angemesssener hier.

Reply:
Mit der Änderung kann ich die halbe Datei überarbeiten. Würde ich dann gerne [zusammen mit](https://github.com/Gnucash/gnucash-docs/pull/169#discussion_r646161746) in einem neuen PR durchführen. Ich finde es wichtiger, dass der Inhalt erst einmal da ist.

Reply:
… oder als separaten Commit die ganze Doku drauf durchsehen.